<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>531</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft JhengHei UI</family>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
*{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(251,102,102, 200), stop:1 rgba(20,196,188, 210));
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>311</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background:rgb(255, 255, 255);
	font-size:20px;
	border: none;border-radius:20px;
	
}

Qlabel{
font-color:208,208,208;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>210</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	color:208,208,208;
	background-color:#FFFFFF;
	font-size:18px;
	border-style:outset;
	border:2px solid rgb(20,196,188);
	border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string>guo</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
	background:#14C4BC;
	border-style:outset;
	border-radius:10px;
	font: 25 14pt '微软雅黑 Light';
	color: rgb(255,255,255);
}

QPushButton:hover{
	background-color: rgb(22,218,208);
	
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>14</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 14pt '微软雅黑 Light';color: rgb(123,115,106);</string>
    </property>
    <property name="text">
     <string>W E L C O M E</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	color:208,208,208;
	background-color:#FFFFFF;
	font-size:18px;
	border-style:outset;
	border:2px solid rgb(20,196,188);
	border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string>123456</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
	background:#14C4BC;
	border-style:outset;
	border-radius:10px;
	font: 25 14pt '微软雅黑 Light';
	color: rgb(255,255,255);
}

QPushButton:hover{
	background-color: rgb(22,218,208);
	
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
	background:#14C4BC;
	border-style:outset;
	border-radius:10px;
	font: 25 14pt '微软雅黑 Light';
	color: rgb(255,255,255);
}

QPushButton:hover{
	background-color: rgb(22,218,208);
	
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}</string>
    </property>
    <property name="text">
     <string>忘记密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_1</tabstop>
  <tabstop>login_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
